Bigtable/src/Table.php

```diff
@@ -120,7 +120,16 @@ public function __construct(
      * @param array $rowMutations An associative array with the key being the
      *        row key and the value being the
      *        {@see \Google\Cloud\Bigtable\Mutations} to perform.
-     * @param array $options [optional] Configuration options.
+     * @param array $options {
+     *     Configuration options.
+     *
+     *     @option RetrySettings|array $retrySettings {
+     *         Only $maxRetries and $totalTimeoutMillis work for RetrySettings in this API.
+     *
+     *         @option int $maxRetries Number of times to retry. **Defaults to** `3`.
+     *         @option int $totalTimeoutMillis The max accumulative timeout in total.
```

Comment:
should we specify in the doc that 0 means no total timeout?

Reply:
We can do that, although it maybe relevant to note that in the other API methods, `RetrySettings::$totalTimeoutMillis` is required, and so setting it to `0` actually results in the value `0` for total timeout (which would essentially disable retries).

For that reason, it probably would make more sense for us to make `-1` mean no total timeout, so that if a user passes in `0` in both cases, the behavior is consistent. What do you think?

Reply:
I know in java 0 means no total timeout

Reply:
Then unless it's a requirement for this feature (which maybe it is?) I'd rather leave it consistent with the rest of our `RetrySettings`